### DISCUSSION TITLE
Generate patient bundles with specific resource types

### DISCUSSION THREAD

**Discussion by AshrafMahamid** [synthetichealth/synthea#1114] [Q&A]
Is it possible to control the resource types the patient could have?

In our case we want to have specific bundle types in some cases and need to guarantee that the generated patients data have a specific resource type, e.g.: MedicationRequest, Observation etc... 

We though about a work around to create X -high number of patients- that could guarantee at least one of them has what we need.
Is the following requested feature: https://github.com/synthetichealth/synthea/issues/1104 provide what we need?

thoughts?
Thanks.

**Comment by jawalonoski** [comment]
> Is it possible to control the resource types the patient could have?

Not directly, no.

> Is the following requested feature: #1104 provide what we need?

Partially. #1104 will ensure that only the resources you specify get exported, but not a guarantee that every patient will have one (or more) of each.

You could try to create a [Keep Module](https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module) to only keep patients with a specific feature, but depending on what you need it could be a complicated conditional statement.

**Reply by AshrafMahamid** [reply]
Thanks, your answer was very helpful!
